parts/k8s/kubernetescustomscript.sh

```diff
@@ -6,6 +6,7 @@ source /opt/azure/containers/provision_installs.sh
 source /opt/azure/containers/provision_configs.sh
 
 CUSTOM_SEARCH_DOMAIN_SCRIPT=/opt/azure/containers/setup-custom-search-domains.sh
+MASTER_INDEX=$(hostname | tail -c 2)
```

Comment:
`MASTER_INDEX` was previously defined at https://github.com/Azure/acs-engine/blob/master/parts/k8s/kubernetesmasterresources.t#L927 and used in the custom script for etcd peer certs (https://github.com/Azure/acs-engine/pull/3769/files#diff-b57d179f607a757733708864ea1bc0cbL44). This is going to overwrite the var so we should consolidate it in one place, probably remove the other one and stick with this new one.

Reply:
Also, I see it is also defined in kubernetes_configs at https://github.com/Azure/acs-engine/pull/3769/files#diff-b57d179f607a757733708864ea1bc0cbR2, this file sources the kuberenetes_configs file so no need to define it here again